api/base.ts

```diff
@@ -0,0 +1,15 @@
+import axios from "axios"
+const base_url = process.env.NEXT_PUBLIC_BASE_URL
+
+export const base = axios.create({
+  baseURL: base_url,
+  headers: { "Content-Type": "application/json" },
+})
```

Comment:
is this expected? The base method is `create`? That means the base request is a POST for all requests

Reply:
I think, yes. Based on the docs. You can specify what type of request you want when calling the base itself.
like this one 
![image](https://user-images.githubusercontent.com/91857581/189047254-5d4e7543-42e1-47aa-a2a0-8fbc090a5710.png)
But I'll check if there's another way this can be done